app/controllers/chat_channel_memberships_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   def find_by_chat_channel_id
     @membership = ChatChannelMembership.where(chat_channel_id: params[:chat_channel_id], user_id: current_user.id).first
+    raise ActiveRecord::RecordNotFound if @membership.blank?
```

Comment:
this can be simplified with `first!` which raises that exception if the item is not found, see https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-first-21

Reply:
updated